download_artifacts.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+
+from coverage_crawler import latest_cov_build
+
+latest_cov_build.download_artifacts()
```

Comment:
We should move the argparse interface from latest_cov_build.py to here too.

Reply:
Ok, I'll update this and the other PR on the weekends. :)